eas.json

```diff
@@ -14,11 +14,21 @@
       "channel": "development"
     },
     "production": {
-      "channel": "production"
+      "channel": "production",
+      "autoIncrement": true
     }
   },
   "submit": {
-    "production": {},
+    "production": {
+      "ios": {
+        "appleId": "$IOS_USER_ID",
+        "appleTeamId": "$IOS_TEAM_ID",
+        "ascAppId": "$IOS_APP_ID"
```

Comment:
does this actually work, i.e. EAS interprets this as a request to read these values from the environment?

Reply:
no, the environment stuff doesnt work sine this config is specifically for the eas command, not part of the app. So my initial approach is how we will need to do it! As per your suggestion putting that into a script vs commands in the action